app/grandchallenge/products/forms.py

```diff
@@ -26,3 +29,13 @@ class ProjectAirFilesForm(SaveFormInitMixin, forms.ModelForm):
     class Meta:
         model = ProjectAirFiles
         fields = ["title", "study_file"]
+
+
+class ProductsPostUpdateForm(PostUpdateForm):
+    class Meta(PostForm.Meta):
```

Comment:
The inheritance of the Meta class here is probably not you want as it's skipping `PostUpdateForm.Meta` and going directly to the super class of that. Probably what you want to do here is replace all insances of `PostForm.Meta` with `PostUpdateForm.Meta` and then the field and widget inheritance becomes much simpler.

Reply:
fixed